utils/build-script

```diff
@@ -40,7 +40,7 @@ from swift_build_support.swift_build_support.SwiftBuildSupport import (
     SWIFT_REPO_NAME,
     SWIFT_SOURCE_ROOT,
     get_all_preset_names,
-    get_preset_options,  
```

Comment:
Nitpick: generally we frown on removing trailing space from an entire file because it muddies up the diff. While we don't want to introduce new ones, can you set your editor not to do it by default?